test/images/resource-consumer/Dockerfile

```diff
@@ -12,7 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jess/stress
+FROM BASEIMAGE
```

Comment:
In the absence of `BASEIMAGE` I'm considering building image for all supported archs like - `amd64, arm, arm64, ppc64le, s390x` and this suits only images based on scratch where we dependent only on the binary image which are packaging.

In the above case I don't want to build this image for `s390x` platform because I'm facing issues while running cross platform using qemu.

So IMO we should have some flexibility to disable any image for any given architecture in the future.